docs/source/developers/java/building.rst

```diff
@@ -195,7 +195,7 @@ CMake
 
 - To build all JNI libraries (MacOS / Linux) except the JNI C Data Interface library:
 
-  .. code-block::
+  .. code-block:: text
```

Comment:
Unfortunately the `console` code block also triggers this warning I tried to fix:

```
/Users/alenkafrim/repos/arrow-new-docs/docs/source/developers/java/building.rst:198: WARNING: Could not lex literal_block "$ cd arrow\n$ brew bundle --file=cpp/Brewfile\nHomebrew Bundle complete! 25 Brewfile dependencies now installed.\n$ brew uninstall aws-sdk-cpp\n(We can't use aws-sdk-cpp installed by Homebrew because it has\nan issue: https://github.com/aws/aws-sdk-cpp/issues/1809 )\n$ export JAVA_HOME=<absolute path to your java home>\n$ mkdir -p java-dist cpp-jni\n$ cmake \\\n    -S cpp \\\n    -B cpp-jni \\\n    -DARROW_BUILD_SHARED=OFF \\\n    -DARROW_CSV=ON \\\n    -DARROW_DATASET=ON \\\n    -DARROW_DEPENDENCY_SOURCE=BUNDLED \\\n    -DARROW_DEPENDENCY_USE_SHARED=OFF \\\n    -DARROW_FILESYSTEM=ON \\\n    -DARROW_GANDIVA=ON \\\n    -DARROW_GANDIVA_STATIC_LIBSTDCPP=ON \\\n    -DARROW_ORC=ON \\\n    -DARROW_PARQUET=ON \\\n    -DARROW_S3=ON \\\n    -DARROW_USE_CCACHE=ON \\\n    -DCMAKE_BUILD_TYPE=Release \\\n    -DCMAKE_INSTALL_LIBDIR=lib/<your system's architecture> \\\n    -DCMAKE_INSTALL_PREFIX=java-dist \\\n    -DCMAKE_UNITY_BUILD=ON\n$ cmake --build cpp-jni --target install --config Release\n$ cmake \\\n    -S java \\\n    -B java-jni \\\n    -DARROW_JAVA_JNI_ENABLE_C=OFF \\\n    -DARROW_JAVA_JNI_ENABLE_DEFAULT=ON \\\n    -DBUILD_TESTING=OFF \\\n    -DCMAKE_BUILD_TYPE=Release \\\n    -DCMAKE_INSTALL_LIBDIR=lib/<your system's architecture> \\\n    -DCMAKE_INSTALL_PREFIX=java-dist \\\n    -DCMAKE_PREFIX_PATH=$PWD/java-dist\n$ cmake --build java-jni --target install --config Release\n$ ls -latr java-dist/lib/<your system's architecture>/*_{jni,java}.*\n|__ libarrow_dataset_jni.dylib\n|__ libarrow_orc_jni.dylib\n|__ libgandiva_jni.dylib" as "console". Highlighting skipped.
```

Reply:
Maybe you need to add `#` comment indicator to the lines with free text?

Reply:
Will try 👍

Reply:
Oh...

Does the following change solve it?

```diff
diff --git a/docs/source/developers/java/building.rst b/docs/source/developers/java/building.rst
index c400b95f2..e8a499d1c 100644
--- a/docs/source/developers/java/building.rst
+++ b/docs/source/developers/java/building.rst
@@ -203,7 +203,7 @@ CMake
       $ brew uninstall aws-sdk-cpp
       (We can't use aws-sdk-cpp installed by Homebrew because it has
       an issue: https://github.com/aws/aws-sdk-cpp/issues/1809 )
-      $ export JAVA_HOME=<absolute path to your java home>
+      $ export JAVA_HOME=${ABSOLUTE_PATH_TO_YOUR_JAVA_HOME}
       $ mkdir -p java-dist cpp-jni
       $ cmake \
           -S cpp \
@@ -221,7 +221,7 @@ CMake
           -DARROW_S3=ON \
           -DARROW_USE_CCACHE=ON \
           -DCMAKE_BUILD_TYPE=Release \
-          -DCMAKE_INSTALL_LIBDIR=lib/<your system's architecture> \
+          -DCMAKE_INSTALL_LIBDIR=lib/${YOUR_SYSTEM_ARCHITECTURE} \
           -DCMAKE_INSTALL_PREFIX=java-dist \
           -DCMAKE_UNITY_BUILD=ON
       $ cmake --build cpp-jni --target install --config Release
@@ -232,11 +232,11 @@ CMake
           -DARROW_JAVA_JNI_ENABLE_DEFAULT=ON \
           -DBUILD_TESTING=OFF \
           -DCMAKE_BUILD_TYPE=Release \
-          -DCMAKE_INSTALL_LIBDIR=lib/<your system's architecture> \
+          -DCMAKE_INSTALL_LIBDIR=lib/${YOUR_SYSTEM_ARCHITECTURE} \
           -DCMAKE_INSTALL_PREFIX=java-dist \
           -DCMAKE_PREFIX_PATH=$PWD/java-dist
       $ cmake --build java-jni --target install --config Release
-      $ ls -latr java-dist/lib/<your system's architecture>/*_{jni,java}.*
+      $ ls -latr java-dist/lib/${YOUR_SYSTEM_ARCHITECTURE}/*_{jni,java}.*
       |__ libarrow_dataset_jni.dylib
       |__ libarrow_orc_jni.dylib
       |__ libgandiva_jni.dylib
```

Reply:
`#` comment indicator to the lines with free text solves the issue 👍